armi/reactor/converters/uniformMesh.py

```diff
@@ -814,21 +761,44 @@ def __init__(self, cs=None, calcReactionRates=True):
         UniformMeshGeometryConverter.__init__(self, cs)
         self.calcReactionRates = calcReactionRates
 
-    def _setParamsToUpdate(self):
-        """Activate conversion of various neutronics-only paramters."""
-        UniformMeshGeometryConverter._setParamsToUpdate(self)
+    def _setParamsToUpdate(self, direction):
+        """
+        Activate conversion of the specified parameters.
+
+        Notes
+        -----
+        The parameters mapped into and out of the uniform mesh will vary depending on
+        the physics kernel using the uniform mesh. The parameters to be mapped in each
+        direction are defined as a class attribute. New options can be created by extending
+        the NeutronicsUniformMeshConverter with different class attributes for parameters to
+        map.
+
+        Parameters
+        ----------
+        direction : str
+            "in" or "out". The direction of mapping; "in" to the uniform mesh assembly, or "out" of it.
+            Different parameters are mapped in each direction.
+        """
+        UniformMeshGeometryConverter._setParamsToUpdate(self, direction)
```

Comment:
This call doesn't do anything, except set up some empty lists. Which is fine. But if you wanted to go the `NotImplementedError` route that I suggested in another comment, then this obviously won't work.

Reply:
I initially thought there might be some parameters that we want to map in all cases that would be defined in the base class implementation. But I can't think of any good examples, so... ?